README.md

```diff
@@ -100,23 +100,53 @@ Django Commons packages.
 
 ## New Project Playbook
 
+Goal: Publish a new release of the project from django-commons repo to PyPI and Test PyPI, using the release workflow.
+
 Assuming the repository name is `repo-name`:
 
+```terraform
+"your-repo-here" = {
+  description = "Your project's description here"
+  allow_merge_commit = true # Allow merge commits on pull requests
+  allow_rebase_merge = true # Allow rebase and merge commits on pull requests
+  allow_squash_merge = true # Allow squash and merge commits on pull requests - Recommended
+  allow_update_branch = true # Allow updating source branch on pull requests
+  has_discussions = false # Enable discussions in project's repository
+  has_wiki = false # Enable wiki in project's repository
+  admins = [
+    # Include people who can release new versions
+    "your-username-here",
+  ]
+  committers = [
+    # Include people who can commit to main / merge changes
+  ]
+  members = [
+    # Include people who can assign/triage tickets
+  ]
+}
+```
+
 ### Pre Transfer Steps
 
 - [ ] Check if the repository meets [inbound requirements][3].
 - [ ] Confirm who will be the admins and maintainers for the repository
 - [ ] Make sure the there are no teams `{repo-name}`, `{repo-name}-admins` and `{repo-name}-committers` in the Django
   Commons organization. Teams can be viewed [here][teams]. The teams will be created by the terraform apply process.
-- [ ] (project owner) PyPI project owner must add the Django Commons PyPI Admins (`cunla`, `stormheg`) as owners in [PyPI][pypi],
-  and [test-pypi][test-pypi]
 - [ ] [Add repository owner to Django Commons as member](#new-member-playbook) (they'll be added to a team later)
-- [ ] (project owner) Transfer the existing repository to the Django Commons organization using the GitHub UI, so old
+
+### Transfer ownership in GitHub, test PyPI and PyPI.
+
+These should be done by the project owner.
+
+- [ ] Transfer the existing repository to the Django Commons organization using the GitHub UI, so old
   information is preserved. See [GitHub docs][gh-docs-transfer-repo].
+    - This step takes time, and therefore it is recommended to do this first.
+- [ ] (project owner) PyPI project owner must add the Django Commons PyPI Admins (`cunla`, `stormheg`) as owners
+  in [PyPI][pypi], and [test-pypi][test-pypi]
 
-### Post Transfer Steps
+### Make GitHub repository managed by terraform
 
-- [ ] Terraform changes to add project to organization
+- [ ] Terraform changes to add project to organization, should be included in the issue opened to transfer the project.
```

Comment:
I have written in my notes to discuss our use of Terraform. For example, most changes normally done through the GitHub UI should now be done using Terraform. Adding new members for example.